src/Helper.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <iostream>
+#include <fstream>
+#include <experimental/filesystem>
+namespace fs = std::experimental::filesystem;
+
+
+
+struct Helper
```

Comment:
The name Helper is not meaningful. The name should suggest what objects of this class are doing.

Reply:
Done